tests/integration/server/users_api_tests.py

```diff
@@ -80,3 +81,37 @@ def test_create_new_user_api():
             "http://rbac-server:8000/api/users/" + user_id
         )
         assert user_details_response.json()["data"]["manager"] == manager_id
+
+
+def test_delete_user_by_next_id():
+    """Test that a user can be deleted by next_id."""
+    user_input = {
+        "name": "Sri Nuthal",
+        "username": "nuthalapati1",
+        "password": "123456",
+        "email": "sri@gmail.com",
+    }
+    expected = {"deleted": 1}
+    with requests.Session() as session:
+        response = session.post("http://rbac-server:8000/api/users", json=user_input)
+        next_id = response.json()["data"]["user"]["id"]
+        # Wait 3 seconds till blockchain process add user.
+        time.sleep(3)
```

Comment:
FYI: Once the "Stabilize integration tests #155" story is completed, these kind of work around are not necessary, anymore. For now, this is fine.